src/Quartz/Impl/StdSchedulerFactory.cs

```diff
@@ -508,11 +516,14 @@ protected virtual void ValidateConfiguration()
 
             Type tpType = loadHelper.LoadType(cfg.GetStringProperty(PropertyThreadPoolType)) ?? typeof(SimpleThreadPool);
 
+#if WINDOWS_THREADPOOL  // TODO (NetCore Port) - We can't use the ClrThreadPool on .NET Core because cross-plat threadpool implementations don't expose all the APIs we need
+                        // Use the SimpleThreadPool there for now and in the future create a new async-friendly threadpool that will work cross-plat.
             if (tpType == typeof(SimpleThreadPool))
```

Comment:
The QueuedTaskScheduler looks useful. We may be able to simplify it some, though, because I think Quartz.Net doesn't need a priority queue. The old threadpools accepted a priority, but that was just the priority for all Quartz tasks relative to other items that might be in the thread pool. The QueuedTaskScheduler's priority queue is for assigning different priorities to tasks coming from Quartz.Net, which I think isn't something the scheduler interfaces support currently.

I'm looking more carefully through QueuedTaskScheduler now and will likely have some more thoughts on it shortly.